modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/AutocompleteUserMVCResourceCommand.java

```diff
@@ -108,7 +111,16 @@ private List<User> _getUsers(
 			Role role = _roleLocalService.getRole(
 				themeDisplay.getCompanyId(), RoleConstants.PUBLICATIONS_USER);
 
-			return _userLocalService.getRoleUsers(role.getRoleId());
+			Set<User> users = new LinkedHashSet<>();
+
+			users.addAll(_userLocalService.getRoleUsers(role.getRoleId()));
+
+			users.addAll(
+				_userLocalService.getInheritedRoleUsers(
+					role.getRoleId(), 0, 20,
+					new UserScreenNameComparator(true)));
```

Comment:
I believe `getInheritedRoleUsers` include all the users that have been assigned the role explicitly. So it should already contains the set from `getRoleUsers`. Can you double check?

Reply:
You're right! I'll update this also